tools/protodoc/protodoc.py

```diff
@@ -767,11 +775,14 @@ def visit_file(self, file_proto, type_context, services, msgs, enums):
         if not has_messages:
             header = ':orphan:\n\n' + header
         warnings = ''
-        if file_proto.options.HasExtension(status_pb2.file_status):
-            if file_proto.options.Extensions[status_pb2.file_status].work_in_progress:
-                warnings += (
-                    '.. warning::\n   This API is work-in-progress and is '
-                    'subject to breaking changes.\n\n')
+        added_wip_warning = False
+        if file_proto.options.HasExtension(udpa_status_pb2.file_status):
+            if file_proto.options.Extensions[udpa_status_pb2.file_status].work_in_progress:
+                added_wip_warning = True
+                warnings += WIP_WARNING
+        if not added_wip_warning and file_proto.options.HasExtension(xds_status_pb2.file_status):
+            if file_proto.options.Extensions[xds_status_pb2.file_status].work_in_progress:
+                warnings += WIP_WARNING
```

Comment:
Do we need any protodoc tests for this?

Reply:
AFAICT there are no protodoc tests. cc @phlax if I am missing them.

I agree having tests would be nice, but I would prefer to not block on that for this change. Can I open a help wanted issue to track?

Reply:
there are currently no tests on landed code.

i have a couple of PRs in the pipeline that would change that

Reply:
Yeah, I think I was confusing with `protoxform` tests. Opening an issue is fine.

Reply:
https://github.com/envoyproxy/envoy/issues/18271